### DISCUSSION TITLE
Feature Request: Augmented Reality Nav Aid

### DISCUSSION THREAD

**Discussion by TheBoatyMcBoatFace** [bareboat-necessities/lysmarine_gen#241] [Ideas]
I was browsing West Marine and came across this. https://www.westmarine.com/raymarine-ar200-augmented-reality-pack-with-cam-210-19418540.html

With BN already having Radar, AIS, cameras, etc... it would be really interesting to have an AR overlay of a video feed. 

Image from product below

![image](https://user-images.githubusercontent.com/29107683/190292460-b22d3ac1-12ba-4dbe-86f4-4bc53e530311.jpeg)

**Comment by mgrouch** [comment]
There is already one augmented reality app built in. Called Stellarium. You can run it by clicking 'Sky' icon.
It will give you augmented reality stars and sky view according to your heading.

**Comment by mgrouch** [comment]
Do you have camera with BBN Marine OS installation on your boat?

**Comment by mgrouch** [comment]
I've created GitHub issue to track this request: https://github.com/bareboat-necessities/lysmarine_gen/issues/242

**Comment by mgrouch** [comment]
There was an attempt in this project

https://github.com/dakk/sailing-cv

sailing computer vision
